packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -231,6 +238,7 @@ export class AtomicSearchBox {
   }
 
   private onInput(value: string) {
+    this.isExpanded = true;
```

Comment:
After submitting a query, the search box is no longer blurred, but suggestions are hidden. I find it a more natural behaviour if typing again shows new suggestions.

Reply:
I agree that's better! Small downsides is you have to type something or blur and come back to see suggestions again.